stack

```diff
@@ -208,6 +208,7 @@ function solver() {
   export STORE_CONN_STR=postgres://postgres:postgres@localhost:5432/solver-db?sslmode=disable
   export STORE_GORM_LOG_LEVEL=silent
   export SERVER_VALIDATION_TOKEN_SECRET=912dd001a6613632c066ca10a19254430db2986a84612882a18f838a6360880e
+  export SERVER_VALIDATION_TOKEN_KID=key-dev
```

Comment:
Thought that tagging the key IDs by network might be a good idea, but open to another naming convention.

Reply:
I agree it's much better, I've updated the jwt-auth to use `key-local`, `key-dev` (using doppler secret in devnet), and `key-test` (using doppler secret in testnet). Thanks Brian!

Reply:
Yeah, having the names will be nice for debugging the tokens! 

We may want to keep `key-dev` for local development to match our naming convention in other places. The names we use for networks are `dev`, `devnet`, `demonet`, and `testnet`. Networks are selected at the CLI using these names, for example `--network devnet` or `--network testnet`, which load an embedded config with a matching name from: https://github.com/Lilypad-Tech/lilypad/tree/main/pkg/options/configs